miio/vacuum.py

```diff
@@ -357,7 +357,7 @@ def timer(self) -> List[Timer]:
         """Return a list of timers."""
         timers = list()
         for rec in self.send("get_timer", [""]):
-            timers.append(Timer(rec))
+            timers.append(Timer(rec, timezone=self.timezone()))
```

Comment:
Move the `self.timezone()` call out from the loop, that will otherwise cause unnecessary I/O.